### ISSUE TITLE
Support appendTo for Autocomplete, Calendar, Colorpicker in PrimeNg

### ISSUE THREAD

**Issue by maitrungduc1410** [udos86/ng-dynamic-forms#1147]
## I'm submitting a

<pre><code>
[ ] Bug / Regression
[x] Feature Request / Proposal
</code></pre>

## I'm using

<pre><code>
NG Dynamic Forms Version: `X.Y.Z`

[ ] Basic UI
[ ] Bootstrap UI  
[ ] Foundation UI
[ ] Ionic UI
[ ] Kendo UI
[ ] Material  
[ ] NG Bootstrap
[x] Prime NG
</code></pre>

## Description

It's not possible to set the `appendTo` on the Calender, Colorpicker and Autocomplete components in PrimeNG

**Comment by ssunil3232** [comment]
Yes, this might resolve this issue with pop-up content from being cut-off.
